lib/src/models/email.dart

```diff
@@ -48,7 +48,7 @@ class Email {
         'from': from.toJson(),
         'reply_to': replyTo?.toJson(),
         'subject': subject,
-        'content': content.map((e) => e.toJson()).toList(),
+        'content': content?.map((e) => e.toJson())?.toList(),
```

Comment:
content field is not need to specify at using template.

Reply:
This looks good, can you move the content parameter optional in the initializer?

Reply:
@dylanshine You're right, I fixed it.

Reply:
@kazu0620 Thank you for your contribution

Version 0.1.2 has been published to https://pub.dev/packages?q=sendgrid_mailer

If you wouldn't mind giving it a star on here and pub.dev, to help other people discover the package.